ustriage/ustriage.py

```diff
@@ -362,9 +367,36 @@ def main(start=None, end=None, debug=False, open_in_browser=False,
     else:
         activitysubscribers = []
     bugs = create_bug_list(
-        start, end, lpname, bugsubscriber, nodatefilter, activitysubscribers
+        date_range['start'], date_range['end'],
+        lpname, bugsubscriber, nodatefilter, activitysubscribers
     )
-    print_bugs(bugs, open_in_browser, shortlinks, blacklist=blacklist)
+    print_bugs(bugs, open_browser['triage'], shortlinks, blacklist=blacklist)
+
+    if expiration['show_expiration']:
+        logging.info('---')
+        logging.info('Expiration check I')
```

Comment:
Can we make this something like "Server Next Expired Bugs" or "Tagged-Next Expired Bugs" rather than "Check I"? This way it is more clear when I copy and paste this for folks who do not look at this every day?